src/libretro/libretro.h

```diff
@@ -2175,6 +2175,12 @@ struct retro_memory_descriptor
  * .len can be implied by .select in many of them, but was included for clarity.
  */
 
+struct retro_memory_descriptor_kodi
```

Comment:
Where's a good place for this? It can't be in libretro.h because this is an upstream file. I should add libretro-common as a depend, import the include from there, and drop this file.